### ISSUE TITLE
v1.7.4  ETA time is negative

### ISSUE THREAD

**Issue by BOBYou** [ava-labs/avalanchego#992]
**Describe the bug**

1. ETA time is negative.

2. How long does it take to synchronize a new node?
I synced for ten days and didn't finish it ( 8 days for v1.7.3,  2 days for v1.7.4).
Is there any way to improve synchronization speed? Or start parameters?

```
INFO [01-20|05:29:37] <C Chain> snow/engine/common/queue/jobs.go#173: executed 750000 of 199784 operations. ETA = -20h10m31.922929906s
INFO [01-20|05:34:33.697] Persisted trie from memory database      nodes=216,529 size=49.47MiB time=4.491865949s gcnodes=1,040,933 gcsize=374.40MiB gctime=3.404207326s livenodes=6154  livesize=-34154057.00B
INFO [01-20|05:39:27] <C Chain> snow/engine/common/queue/jobs.go#173: executed 752500 of 199784 operations. ETA = -20h19m13.393788799s
INFO [01-20|05:40:50.232] Persisted trie from memory database      nodes=163,718 size=39.13MiB time=3.107166383s gcnodes=749,664   gcsize=270.54MiB gctime=2.379763399s livenodes=7164  livesize=-33904727.00B
INFO [01-20|05:40:50.240] committed atomic trie                    c=atomicTrie hash=0x686686819bb76ef711d90874a3e75cdb25c01b70539fd18de58308744b967826 height=8,171,520
INFO [01-20|05:47:51] <C Chain> snow/engine/common/queue/jobs.go#173: executed 755000 of 199784 operations. ETA = -20h26m51.563423174s
INFO [01-20|05:54:33.691] Persisted trie from memory database      nodes=310,626 size=67.01MiB time=5.826573168s gcnodes=1,800,166 gcsize=648.44MiB gctime=6.157038415s livenodes=6247  livesize=-34408626.00B
INFO [01-20|05:54:33.695] committed atomic trie                    c=atomicTrie hash=0x6c8f14c53c876b0f5d35b2988c0473875b49951403e62e0a488f8bd53e9fd68f height=8,175,616
INFO [01-20|05:55:54.327] Persisted trie from memory database      nodes=51079   size=13.89MiB time=900.169916ms gcnodes=148,182   gcsize=53.82MiB  gctime=397.933176ms livenodes=4236  livesize=-35161640.00B
INFO [01-20|05:56:10] <C Chain> snow/engine/common/queue/jobs.go#173: executed 757500 of 199784 operations. ETA = -20h34m26.118041239s
```

```
INFO [01-20|03:18:14] <C Chain> snow/engine/common/queue/jobs.go#173: executed 712500 of 199784 operations. ETA = -18h12m51.575930366s
INFO [01-20|03:19:17.873] Persisted trie from memory database      nodes=103,975 size=25.81MiB time=1.819930657s gcnodes=434,257   gcsize=156.66MiB gctime=1.291007925s livenodes=5631  livesize=-32529230.00B
INFO [01-20|03:24:17] <C Chain> snow/engine/common/queue/jobs.go#173: executed 715000 of 199784 operations. ETA = -18h18m42.098751673s
INFO [01-20|03:27:34.988] Persisted trie from memory database      nodes=223,639 size=50.11MiB time=4.105637279s gcnodes=1,205,703 gcsize=436.18MiB gctime=4.032081113s livenodes=6273  livesize=-32450961.00B
INFO [01-20|03:27:34.992] committed atomic trie                    c=atomicTrie hash=0xc970daedaf509c2da3e79ea483fedacf76c51577754d4bf1a9d7395cb439ca3d height=8,134,656
INFO [01-20|03:32:32] <C Chain> snow/engine/common/queue/jobs.go#173: executed 717500 of 199784 operations. ETA = -18h26m8.780507631s
INFO [01-20|03:40:05.707] Persisted trie from memory database      nodes=255,543 size=56.44MiB time=4.83651937s  gcnodes=1,403,569 gcsize=507.81MiB gctime=4.723944055s livenodes=7042  livesize=-32267106.00B
INFO [01-20|03:42:20] <C Chain> snow/engine/common/queue/jobs.go#173: executed 720000 of 199784 operations. ETA = -18h34m42.476533963s
INFO [01-20|03:42:56.786] Persisted trie from memory database      nodes=86568   size=21.84MiB time=1.516012852s gcnodes=301,583   gcsize=110.55MiB gctime=897.242419ms livenodes=5373  livesize=-32938492.00B
INFO [01-20|03:42:56.791] committed atomic trie                    c=atomicTrie hash=0x48b63c91dbc99a7c46b6ec5621a5c12a201e81cd15a6b1045fdce00fd26f9e52 height=8,138,752
```

$ ./avalanchego --version
avalanche/1.7.4 [database=v1.4.5, commit=ca7a521aa6c5a49c1fb52ef1e7e0ddaad6a0e760]

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal

**Comment by BOBYou** [comment]
everybody,
～/.avalanchego/db/mainnet/v1. 4.5/
How much space does this directory take up?

$ du -h ~/.avalanchego/db/mainnet/v1.4.5/
103G    /home/ubuntu/.avalanchego/db/mainnet/v1.4.5/

**Comment by rickcain** [comment]
@BOBYou Currently db directory takes about 160G.

**Comment by BOBYou** [comment]
@rickcain  thanks

Finally, the synchronization succeeded!  18 days 

176G    .avalanchego/db/mainnet/v1.4.5

**Comment by StephenButtolph** [comment]
This issue will be resolved in v1.7.5. Thanks for raising it here.
